src/components/FormControlLabel/.stylelintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../.stylelintrc.json",
+  "rules": {
+    "custom-property-pattern": "(denhaag|utrecht)-[a-z0-9-]+",
```

Comment:
Wouldn't it be better to just allow this for more components? I don't think this will be the only component which will use utrecht tokens.